assgnment2/funcions.dart

```diff
@@ -0,0 +1,183 @@
+
```

Comment:
Rename the file to be more meaningful or at least fix the typo (it should be `functions`)